core/esmf-aspect-meta-model-java/src/test/java/org/eclipse/esmf/aspectmodel/loader/ValueInstantiatorTest.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright (c) 2025 Robert Bosch Manufacturing Solutions GmbH
+ *
+ * See the AUTHORS file(s) distributed with this work for additional
+ * information regarding authorship.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ *
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+package org.eclipse.esmf.aspectmodel.loader;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.util.Locale;
+import java.util.Optional;
+
+import org.eclipse.esmf.metamodel.ScalarValue;
+import org.eclipse.esmf.metamodel.datatype.LangString;
+
+import org.apache.jena.rdf.model.ResourceFactory;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+class ValueInstantiatorTest {
+
+   @BeforeAll
+   static void setupJena() {
+      ResourceFactory.createResource();
+   }
+
+   @Test
+   void testBuildLanguageStringEmptyLanguageTag() {
+      // Arrange
+      String lexicalRepresentation = "hello";
+      String languageTag = "";
+      String datatypeUri = "http://www.w3.org/1999/02/22-rdf-syntax-ns#langString";
```

Comment:
Instead of the constant, please refer to the `org.apache.jena.vocabulary.RDF` vocabulary:

```suggestion
      String datatypeUri = RDF.langString.getURI();
```

Reply:
Fixed!